### PR TITLE
feat(polymer): add polymer install and build

### PR DESCRIPTION
**Pull request by dmccaffery**
* automatically detect polymer.json
* detect polymer-cli and bower-cli from node_modules if not installed globally
* execute npm install
* execute bower install
* execute polymer install
* execute polymer build
* copy polymer output to artifacts/build
* add test project for npm, bower, and polymer-cli

Closes: #105

### CODE REVIEW

File: test/AM.Condo.FrontEnd.Test/src/my-app.html [hunk 1]
```diff
@@ -0,0 +1,166 @@
...
+
+      static get observers() {
+        return [
+          '_routePageChanged(routeData.page)',
+        ];
+      }
+
+      _routePageChanged(page) {
+        // If no page was found in the route data, page will be an empty string.
+        // Deault to 'view1' in that case.
```
Comment (tiffanywang3) [inline test/AM.Condo.FrontEnd.Test/src/my-app.html]: Default

Comment (dmccaffery) [reply test/AM.Condo.FrontEnd.Test/src/my-app.html]: This was autogenerated code from the polymer-cli for a minimally required app in order for ```polymer build``` to work... its here in condo for the sole purpose of making sure that condo executes the CLI command correctly and that it works.

**Review by tiffanywang3 — DISMISSED** [review]
cheeky comment but lgtm!

File: test/AM.Condo.FrontEnd.Test/src/my-app.html (hunk 1 above)
Comment (dmccaffery) [reply test/AM.Condo.FrontEnd.Test/src/my-app.html]: Fixed.

**Pull request merged**